### ISSUE TITLE
Getting less Accuracy on swin tiny ADE20k

### ISSUE THREAD

**Issue by ChakradarReddy** [Picsart-AI-Research/SeMask-Segmentation#15]
In SeMask-T FPN while training we are getting an accuracy of 11 .88 mIoU using a single GPU. We know that your code was trained on 2 GPUs. We expected somewhat near to your results at least a score of 30 mIoU. 

We are using NVIDIA RTX A6000 48 GB GPU.

Here is our training command

`python3 tools/train.py configs/semask_swin/ade20k/semfpn_semask_swin_tiny_patch4_window7_512x512_80k_ade20k.py  --options model.backbone.use_checkpoint=True`

Here is our log file
[semask.txt](https://github.com/Picsart-AI-Research/SeMask-Segmentation/files/8142157/semask.txt)

**Comment by praeclarumjj3** [comment]
Hi, thanks for your interest in our work. It seems to me that you are not loading the pretrained `swin-tiny model` when training the `SeMask-T FPN` network. You need to pass the path of the checkpoint as an argument as well:

```.bash
python3 tools/train.py configs/semask_swin/ade20k/semfpn_semask_swin_tiny_patch4_window7_512x512_80k_ade20k.py --options model.pretrained=swin_tiny_patch4_window7_224.pth
```

You can find the pre-trained backbone model on the official [Swin-Transformer repo](https://github.com/microsoft/Swin-Transformer#main-results-on-imagenet-with-pretrained-models).

**Comment by jaideep19** [comment]
Thank you so much @praeclarumjj3
